draft-ietf-cose-cbor-encoded-cert.md

```diff
@@ -755,18 +755,17 @@ In natively signed requests (types 0 and 2), a positive extensionID is used. In
 
 Enrollment over Secure Transport (EST, {{RFC7030}}) defines, and {{I-D.ietf-lamps-rfc7030-csrattrs}} clarifies, how an EST server can specify what it expects the EST client to include in a subsequent Certificate Signing Request (CSR). Alternatively to the unstructured mechanism specified in {{RFC7030}}, {{Appendix B of RFC8295}} describes an approach using a CSR template: In response to GET /csrattrs by the EST client, the EST server returns an entire CSR object with various fields filled out, and other fields waiting to be filled in by the EST client.
 
-For C509 we follow the approach of {{RFC8295}}. The C509CertificateRequestTemplate is based on TBSCertificateRequest of the C509CertificateRequest, see {{fig-C509CSRCDDL}}, which excludes the subjectSignatureValue field from the template since that needs no further specification.
```

Comment:
Why did you remove this text: "which excludes the subjectSignatureValue field".? That text explains the difference between the format of the Certificate Request and the format of the Template.